zinit-install.zsh

```diff
@@ -1354,7 +1354,7 @@ builtin source "${ZINIT[BIN_DIR]}/zinit-side.zsh" || { builtin print -P "${ZINIT
     } elif [[ -n ${ZINIT_SICE[$id_as]} ]] {
         +zinit-message "{error}WARNING:{msg2} Inconsistency #3" \
             "occurred, please report the string: \`{obj}${ZINIT_SICE[$id_as]}{msg2}' to the" \
-            "GitHub issues page: {obj}https://github.com/zdharma-continuum/zinit/issues/{msg2}.{rst}"
```

Comment:
This should be reversed.

Reply:
Woops..! On it.

Reply:
Done: 95d1a2fb44e416dbebed744fb5c7b8bbf61184e5